modules/test/playwright/tests/export-import-web/export.spec.ts

```diff
@@ -79,6 +80,112 @@ test('cannot export site scoped custom object entries at instance level', async
 	await expect(page.getByLabel('Tests 1 Items')).toBeHidden();
 });
 
+test('can export custom object entries at instance level with date filter', async ({
+	apiHelpers,
+	companyExportImportPage,
+}) => {
+	const objectActionApiClient =
+		await apiHelpers.buildRestClient(ObjectDefinitionApi);
+
+	const {body: objectDefinition} =
+		await objectActionApiClient.postObjectDefinition({
+			active: true,
+			externalReferenceCode: 'test',
+			label: {
+				en_US: 'Test',
+			},
+			name: 'Test',
+			objectFields: [
+				{
+					DBType: ObjectField.DBTypeEnum.String,
+					businessType: ObjectField.BusinessTypeEnum.Text,
+					indexed: true,
+					indexedAsKeyword: true,
+					label: {
+						en_US: 'Name',
+					},
+					name: 'name',
+					required: true,
+				},
+			],
+			pluralLabel: {
+				en_US: 'Tests',
+			},
+			portlet: true,
+			scope: 'company',
+			status: {
+				code: 0,
+			},
+		});
+
+	apiHelpers.data.push({id: objectDefinition.id, type: 'objectDefinition'});
+
+	await apiHelpers.objectEntry.postObjectEntry(
+		{externalReferenceCode: '', name: 'test'},
+		'c/tests'
+	);
+
+	const endDate1 = new Date();
+
+	const startDate1 = new Date();
+
+	startDate1.setDate(startDate1.getDate() - 1);
+
+	const exportFilePath1 = await companyExportImportPage.export(
+		'Tests 1 Items',
+		false,
+		toDateRangeDate(endDate1),
+		'',
+		'',
```

Comment:
This triggers me a little bit. I can suggest creating an structure here like:
```
companyExportImportPage.export(
		'Tests 1 Items',
		false,
		{
			endDate: toDateRangeDate(endDate2),
			endTime: toDateRangeTime(endDate2),
			startDate: toDateRangeDate(startDate2),
			startTime: toDateRangeTime(startDate2)
		}
	);
```
Just to get rid of the optional fields if they are not needed